src/Reflection/ReflectionFunction.php

```diff
@@ -102,4 +102,16 @@ public function isDisabled() : bool
     {
         return false;
     }
+
+    /**
+     * Note - we cannot reflect on internal functions (as there is no PHP source
+     * code we can access. This means, at present, we can only EVER return null
```

Comment:
"can't" = fail = throw, no?

Reply:
I think that `NULL` is valid return value for non-internal functions.

The same comment is already there for some more methods: https://github.com/Roave/BetterReflection/blob/master/src/Reflection/ReflectionFunctionAbstract.php#L287

Reply:
Yeah, but is this viable also for classes that we know to be internal? Shouldn't we throw there?

Reply:
I'm probably a little dumb today :) I'm sorry I don't get how you mean it.

Reply:
A `ReflectionFunction` that is defined and is internal can be reflected on the fly, IMO